app/main.py

```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for character in phrase:
+        if character == letter:
```

Comment:
The comparison here is case sensitive, but the task description specifies that the function should be case insensitive. You might want to convert both 'character' and 'letter' to the same case before comparing them.